triekeyhash_test.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	verkleutils "github.com/ethereum/go-ethereum/trie/utils"
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/require"
+)
+
+func BenchmarkHashAddress(b *testing.B) {
+	addrBytes := make([]byte, common.AddressLength)
+	_, err := rand.Read(addrBytes[:])
+	require.NoError(b, err)
+
+	b.Run("keccak", benchKeccak(addrBytes))
+	b.Run("pedersen hash", func(b *testing.B) {
+		var res []byte
+		// Warmup go-ipa generating precomputes before main bench.
+		res = verkleutils.GetTreeKeyBalance(addrBytes)
```

Comment:
The analogous ones should have a similar cost since it only changes some vector values and not the main logic, so I think this is representative.